src/AudioManagerImpl.test.js

```diff
@@ -37,39 +37,41 @@ function logTuning (noteTable: Array<Array<string>>) {
 }
 
 test("Returns a sensible note range for every supported character position.", () => {
-    // 4 rows "in bounds" on either side of the centre plus testing three "out of bounds" rows on each side.
-    const minRow = -7;
-    const maxRow = 7;
+    // We now have 17 rows and no "out of bounds".
+    const minRow = 1;
+    const maxRow = 16;
 
     // 9 columns "in bounds" above and below the centre plus testing three "out of bounds" columns on each side.
-    const minCol = -12;
-    const maxCol = 12;
+    const minCol = 1;
+    const maxCol = 26;
 
     // noteTable [row][col] = singlePitchString;
     const noteTable = [];
 
-    const sceneDimensions = new SceneDimensions(1, 1);
+    const sceneDimensions = new SceneDimensions(17, 27);
```

Comment:
Is `(17, 27)` what we want here, rather than `(26,16)`? What is the relationship between this `sceneDimensions` object and the `minRow`, `maxRow`, `minCol`, `maxCol` variables above?

Does this code become clearer when the [C2LC-372](https://issues.fluidproject.org/browse/C2LC-372) `SceneDimensions` refactor is done? Would it then make sense to use `sceneDimensions.minX` and such, rather than creating separate variables for the loop bounds?

Reply:
As discussed in meetings, I plan to merge this with C2LC-372 and update to use the new variables.  I'll also generalise the comments to indicate that the width is pulled from the scene dimensions.